create_dmg.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$SHOULD_BUILD" == "yes" ]]; then
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    cd VSCode-darwin
+    create-dmg ../VSCodium-darwin-${LATEST_MS_TAG}.app
```

Comment:
I tried to follow your lead for this bashfile.

Reply:
I think this should be `create-dmg VSCodium.app ..`

The syntax is `create-dmg <src> [dst]` and you want the generated `dmg` file as a sibling to the `zip` file